webapi/src/test/scala/org/knora/webapi/e2e/v2/ResponseCheckerR2RV2.scala

```diff
@@ -209,13 +177,10 @@ object ResponseCheckerR2RV2 {
       * @param receivedJSONLD the received response as JSONLD.
       */
     def compareJsonForMappingCreationResponse(expectedJSONLD: String, receivedJSONLD: String): Unit = {
+        val expectedJsonLDDocument = JsonLDUtil.parseJsonLD(expectedJSONLD)
+        val receivedJsonLDDocument = JsonLDUtil.parseJsonLD(receivedJSONLD)
 
-        val expectedResponseAsScala: Map[IRI, Any] = JSONToScala(expectedJSONLD)
-
-        val receivedResponseAsScala: Map[IRI, Any] = JSONToScala(receivedJSONLD)
-
-        assert(expectedResponseAsScala == receivedResponseAsScala, "Mapping creation request response did not match expected response")
-
+        assert(expectedJsonLDDocument == receivedJsonLDDocument, "Mapping creation request response did not match expected response")
```

Comment:
Should we add a test to make sure this method works correctly?

We could add a test the expects this method to fail. I am thinking of the `received == received` or `expected == expcted` bug ...

Reply:
I know there is no logical way to rule out errors. So I won't ask for a test that checks the tests for the test ...